app/models/manageiq/providers/kubevirt/inventory/parser.rb

```diff
@@ -123,6 +123,8 @@ def process_vm_instance(object)
     vm_object = process_domain(object.namespace, object.memory, object.cpu_cores, uid, name)
     process_status(vm_object, object.ip_address, object.node_name)
 
+    vm_object.host = host_collection.lazy_find(object.node_name, :ref => :by_name)
```

Comment:
:+1: nice, this would be a good enhancement on its own if you want to pull it out, maybe add a spec test to show that it finds the host properly.

Reply:
https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/262

Reply:
:+1: perfect thanks